dnf/base.py

```diff
@@ -1064,6 +1066,15 @@ def download_packages(self, pkglist, progress=None, callback_total=None):
             percent = 100 - real / full * 100
             logger.info(msg, full / 1024 ** 2, real / 1024 ** 2, percent)
 
+        if self.conf.destdir:
+            dnf.util.ensure_dir(self.conf.destdir)
+            for pload in payloads:
+                shutil.copy(pload.pkg.repo.pkgdir + "/" + os.path.basename(pload.pkg.location), self.conf.destdir)
```

Comment:
The line is too long. Merging of two path can be handled by os.path.join(pload.pkg.repo.pkgdir, os.path.basename(pload.pkg.location))